samza-core/src/main/java/org/apache/samza/container/grouper/stream/SSPGrouperProxy.java

```diff
@@ -42,21 +44,21 @@
 /**
  * Provides a stream expansion aware task to partition assignments on top of a custom implementation
  * of the {@link SystemStreamPartitionGrouper}.
+ * @see <a href="https://cwiki.apache.org/confluence/display/SAMZA/SEP-5%3A+Enable+partition+expansion+of+input+streams">SEP-5</a>
```

Comment:
FYI, the Apache URL shortener: https://s.apache.org

Reply:
The url shortener requires an apache.org password. My apache JIRA or confluence passwords don't work. I guess you have to be a committer or PMC to have the credentials.